docs/content/preview/explore/cluster-management/point-in-time-recovery-ycql.md

```diff
@@ -100,7 +100,7 @@ Create and populate a table, get a timestamp to which you'll restore, and then w
 1. Create a snapshot schedule for the new `pitr` keyspace from a shell prompt. In the following example, the schedule is one snapshot every minute, and each snapshot is retained for ten minutes:
 
     ```sh
-    ./bin/yb-admin create_snapshot_schedule 1 10 ycql.pitr
+    ./bin/yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> create_snapshot_schedule 1 10 ycql.pitr
```

Comment:
The angle brackets are definitely going to make all of these commands fail! Please test out commands before you add them to docs :)

```suggestion
    ./bin/yb-admin -master_addresses ip1:7100,ip2:7100,ip3:7100 create_snapshot_schedule 1 10 ycql.pitr
```